Tests/Creation/Frame/Content/String/ID3CommentTypesFrameCreatorTest.swift

```diff
@@ -25,13 +31,20 @@ class ID3CommentTypesFrameCreatorTest: XCTestCase {
                 content: "test"
         )
 
-        #warning("this is going to fail because I don't know how to adapt it for the new type")
-        XCTAssertEqual(frameBytes, [0x22, 0x11, 0x00, /* insert .und in bytes*/ /* insert "description in bytes followed by null character*/ 0x01, 0x00, 0x00, 0x74, 0x65, 0x73, 0x74, 0x00, 0x00])
-    }
-}
-
-class MockCommentTypesStringToBytesAdapter: StringToBytesAdapter {
-    func adapt(string: String, for version: ID3Version) -> [UInt8] {
-        return [0x01, 0x00, 0x00, 0x74, 0x65, 0x73, 0x74, 0x00, 0x00]
+        XCTAssertEqual(
+          frameBytes,
+          [
+            0x22, // Identifier (See above.)
+            0x00, 0x00, 0x00, 0x2D, // Size
+            0x01, // UCS‐2
+            0xFF, 0xFE, 0x75, 0x00, 0x6E, 0x00, 0x64, 0x00, // “und”
+            0xFF, 0xFE, 0x64, 0x00, 0x65, 0x00, 0x73, 0x00, 0x63, 0x00, 0x72, 0x00, 0x69, 0x00, 0x70, 0x00, 0x74, 0x00, 0x69, 0x00, 0x6F, 0x00, 0x6E, 0x00, 0x00, 0x00, // “description”
+            0xFF, 0xFE, 0x74, 0x00, 0x65, 0x00, 0x73, 0x00, 0x74, 0x00 // “test”
+          ]
+        )
+        #warning("Why is the size four bytes? The specification says it should be three.")
```

Comment:
Probably because I was messing with something I didn't really understand. I copied this test from another FrameCreatorTest (I don't remember which one) and didn't touch the parts I didn't understand. So I didn't change the size.